### PR TITLE
add logo for strudel.eth space

### PR DESCRIPTION
**Pull request by johannbarbie**
#### Hi! What is your PR about?

- [x] Add images for your space
- [x] Add custom domain for your space
- [ ] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/strudel.eth/domains.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "snapshot.strudel.finance": "strudel.eth"
+}
```
Comment (gokaiorg) [inline spaces/strudel.eth/domains.json:3]: - You need to add your subdomain in the list of the file https://github.com/snapshot-labs/snapshot-spaces/blob/master/spaces/domains.json
- You should not add domains.json files in your space, please delete it.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/strudel.eth/domains.json (hunk 1 above)
Comment (johannbarbie) [reply spaces/strudel.eth/domains.json:3]: thx, will fix that

**Comment by johannbarbie** [comment]
@gokaiorg i've addressed your comment. how to continue, will you reevaluate, or should i open a new PR?

**Comment by johannbarbie** [comment]
@gokaiorg ping :)

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**